src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -427,6 +427,21 @@ public boolean addReaction(String channelId, String timestamp, String emojiName)
         return postToSlack("reactions.add", json);
     }
 
+    /**
+     * Implementation of {@link SlackService#removeReaction(String, String, String)}
+     */
+    @Override
+    public boolean removeReaction(String channelId, String timestamp, String emojiName) {
+        JSONObject json = SlackReactionRequest.builder()
+                .withChannelId(channelId)
+                .withTimestamp(timestamp)
+                .withEmojiName(emojiName)
+                .build()
+                .getBody();
+
+        logger.fine("Removing reaction:  " + json.toString());
+        return postToSlack("reactions.remove", json);    }
```

Comment:
formatting is off here

Reply:
fix with https://github.com/jenkinsci/slack-plugin/pull/769/commits/b7f18759113465f2858e7f6f44a661950d615f3e